src/index.html

```diff
@@ -17,3 +17,32 @@ <h1 class="logo">MA</h1>
     <script src="scripts/main.js"></script>
   </body>
 </html>
+<script>
+  const logo = document.querySelector('.logo');
+
+  const promise1 = new Promise((resolve) => {
+    logo.addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
+  });
+
+  const promise2 = new Promise((_, reject) => {
+    setTimeout(() => {
+      reject('Promise was rejected!');
+    }, 3000);
+  });
+
+  promise1.then((message) => {
+    const div = document.createElement('div');
+    div.className = 'message';
+    div.textContent = message;
+    document.body.appendChild(div);
+  });
+
+  promise2.catch((message) => {
+    const div = document.createElement('div');
+    div.className = 'message error-message';
+    div.textContent = message;
+    document.body.appendChild(div);
+  });
```

Comment:
When appending elements to the DOM inside a `.catch()` or `.then()`, consider checking if the element already exists to avoid duplicate messages being appended to the `body` on multiple rejections or resolutions.